src/navigator.js

```diff
@@ -174,16 +174,17 @@ $.Navigator = function( options ){
         clickHandler:    $.delegate( this, onCanvasClick ),
         releaseHandler:  $.delegate( this, onCanvasRelease ),
         scrollHandler:   $.delegate( this, onCanvasScroll )
-    }).setTracking( true );
+    });
```

Comment:
This is going to conflict with #575; maybe merge that patch into this one, and we can merge #575 first?